api/yelp_beans/routes/tasks.py

```diff
@@ -104,3 +108,39 @@ def clean_user_subscriptions():
             logging.info(f"Deleted UserSubscriptionPreference<{preference.id}>")
 
     return "OK"
+
+
+@tasks.route("/generate_meeting_requests_for_auto_opt_in_preferences", methods=["GET"])
+def generate_meeting_requests_for_auto_opt_in_preferences():
+    specs = get_specs_for_current_week()
+
+    logging.info("All specs: ")
+    logging.info(specs)
+
+    for spec in specs:
+       spec_dt = get_meeting_datetime(spec)
```

Comment:
My previous suggestion was off by one space I think.
```suggestion
        spec_dt = get_meeting_datetime(spec)
```